Makefile

```diff
@@ -36,22 +39,37 @@ ifdef CUSTOM_CA
 	cp -f ${CUSTOM_CA} docker/conf/certificates/
 endif
 
+# Builds the vagrant box and the example/standalone
 up-standalone: update-box custom_ca
 ifeq ($(GITHUB_ACTIONS),true) # Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.
 	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--extra-vars "\"ci_test=true mode=standalone\""' vagrant up --provision
 else
 	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "\"mode=standalone\""' vagrant up --provision
 endif
 
-up: update-box custom_ca
+# Builds the vagrant box and the example/standalone_git
+# Remember the parameters for GIT integration explained in /example/standalone_git/README.md
+# Example: make up repo=<GitHub-repository> branch=<branch to checkout and track> user=<GitHub username> token=<personal token from GitHub>
+up: check-params update-box custom_ca
 ifeq ($(GITHUB_ACTIONS),true) # Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.
-	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--extra-vars "\"ci_test=true mode=standalone_git\""' vagrant up --provision
+	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--extra-vars "\"ci_test=true mode=standalone_git repo=$(repo) branch=${branch} user=${user} token=${token}\""' vagrant up --provision
 else
-	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "\"mode=standalone_git\""' vagrant up --provision
+	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "\"mode=standalone_git repo=$(repo) branch=${branch} user=${user} token=${token}\""' vagrant up --provision
 endif
 
+# Checks that all parameters are set.
+check-params:
```

Comment:
Make test is working without asking parameters
 m77875@u1804i-0233:~/Projects/Minerva2/terraform-nomad-nifi$ make test
vagrant destroy -f
==> default: VM not created. Moving on...
rm -rf ./tmp
rm -rf ./.vagrant
rm -rf ./.minio.sys
rm -rf ./example/.terraform
rm -rf ./example/terraform.tfstate
rm -rf ./example/terraform.tfstate.backup
==> default: Checking for updates to 'fredrikhgrelland/hashistack'
    default: Latest installed version: 0.9.0
    default: Version constraints: >= 0.9, < 0.10
    default: Provider: virtualbox

Reply:
Are you sure you are running from the right branch? issue_20_manage_secrets

Reply:
I am testing on  this branch issue_50_manage_secrets, now its working